kadai1/yossy0806/cmd/imgconv/main.go

```diff
@@ -0,0 +1,82 @@
+package main
+
+import (
+	"errors"
+	"flag"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	imgconv "github.com/yossy0806/gopherdojo-studyroom/kadai1/yossy0806/imgconv"
```

Comment:
```suggestion
	"github.com/yossy0806/gopherdojo-studyroom/kadai1/yossy0806/imgconv"
```

すでに（パスの一番後ろが）imgconvなので、別名を宣言する必要はないです。別の名前（例: conv）で使いたい場合に宣言します。